README.md

```diff
@@ -17,17 +17,20 @@ Vue.use(components);
 ```
 
 ## Development
-```
+This repo was initialized using npm v7 which changes the `package-lock.json` file format changed significantly. Before installing node modules, make sure you have npm v7 installed.
+
+```bash
+npm install -g npm@latest
```

Comment:
Do we want to fix this at v7.13.0 for now?

Pros: if a v8 (or 9 or what have you) comes out that changes this again and we haven't upgraded yet and a new person comes along, they won't have issues.
Cons: it'll require us to manually change this in the docs if/when we do upgrade.

Unless we just set the expectation that we'll always keep our npms up-to-date, and then the first person to use an upgraded version just commits the new lockfile, though that seems mildly? fragile.

I don't know if I have a preference for any of these, just throwing out thoughts.